bridge_adaptivity/module/tests/test_tasks.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
+from django.test import TestCase
+from mock import patch
+
+from bridge_lti.models import BridgeUser
+from module.models import Activity, Collection, CollectionGroup, Engine
+from module.tasks import sync_collection_engines
+
+
+class TestTask(TestCase):
```

Comment:
you could inherit this class from ```module/tests/test_views.py:BridgeTestCase```.
in this class method setup already implemented all lines which you wrote here.

pls take a look into this class. 

btw maybe we should place it not in ```module/tests``` but somewhere in general folder and re-name it?

Reply:
The inheritance does not make sense, in setUp collection creation need to be avoided and added only to the test.